modules/statistics/AvgRTPStatsReporter.js

```diff
@@ -564,14 +567,22 @@ export default class AvgRTPStatsReporter {
      */
     _calculateAvgStats(data) {
 
+        if (!data) {
+            logger.error('No stats');
+
+            return;
+        }
+
+        if (!data.transport || !data.transport.length) {
+            logger.error('No transport.');
```

Comment:
I see, I'll remove that then. The motivation behind this error/return was that the transport is a determining factor for all the stats that are being logged and we shouldn't mix TCP with UDP. We can adjust the queries to explicitly include the transport type, so even if we log stats without transport, we can ignore them (for now at least).